app/src/jembi/res/values/strings.xml

```diff
@@ -0,0 +1,548 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
+
+    <string name="app_name">Jembi App Store</string>
```

Comment:
@patience-mpofu note that there are many strings.xml files for other languages. because this is just a demo of the white-labelling capabilities, it's OK just to change the en file, but I want to point out that we'd need to (at least consider) changing all the other files too.